src/tbp/monty/frameworks/models/evidence_matching/hypotheses_updater.py

```diff
@@ -219,6 +226,10 @@ def update_hypotheses(
             return []
 
         hypotheses_updates = []
+        telemetry: dict[str, Any] = {}
```

Comment:
Is `Any` the most specific we can get for the values in this dictionary?

Reply:
I'm not sure, I just copied this from https://github.com/thousandbrainsproject/tbp.monty/blob/f0e181c4f8f66103d941df89f622db29aee840fc/src/tbp/monty/frameworks/models/evidence_matching/resampling_hypotheses_updater.py#L277